docs/guide/personalization/recommendation_client.md

```diff
@@ -90,6 +90,18 @@ ibexa:
 | `included_item_types`             | A list of alphanumerical identifiers of item types on which the tracking script is shown. |
 | `random_item_types`               | A list of alphanumerical identifiers of item types that are returned when the response from the server contains no content. |
 
+You can use an alphanumeric Content identificator `remoteId` instead of a numeric `id`. To enable it, add the following code to the configuration file:
```

Comment:
```suggestion
You can use an alphanumeric content identifier `remoteId` instead of a numeric `id`. To enable it, add the following code to the configuration file:
```